pkgs/top-level/all-packages.nix

```diff
@@ -1741,6 +1741,10 @@ in
 
   inherit (ocamlPackages) dune dune_2;
 
+  dune-release = ocamlPackages.callPackage ../development/tools/ocaml/dune-release {
```

Comment:
This should be moved inside `ocamlPackages` or receive it as argument.

Reply:
I guess it makes most sense to have it in `ocamlPackages`, since it exposes some if not all of its modules.